install/installation-redhat.md

```diff
@@ -123,93 +128,129 @@ instead.
 
     </terminal>
 1.  Update your local repository list:
+
     ```bash
-    yum update
+    sudo yum update
     ```
+
 1.  Install TimescaleDB:
+
     ```bash
-    yum install timescaledb-2-postgresql-14
+    sudo yum install timescaledb-2-postgresql-14
     ```
 
+     <highlight type="note">
+     When installing on CentOS 8 or Red Hat Enterprise Linux 8 you need
+     to disable the built-in PostgreSQL module in the system using the
+     `sudo dnf -qy module disable postgresql`command.
+
+</highlight>
+
+1.  Initialize the database:
+
+    ```bash
+    sudo /usr/pgsql-14/bin/postgresql-14-setup initdb
+
 </procedure>
 
 When you have completed the installation, you need to configure your database so
 that you can use it. The easiest way to do this is to run the `timescaledb-tune`
-script, which is included with the `timescaledb-tools` package. For more
+script, which is included with the `timescaledb-tools` package. Run the
+`timescaledb-tune` script using the
+`sudo timescaledb-tune --pg-config=/usr/pgsql-14/bin/pg_config` command. For more
 information, see the [configuration][config] section.
 
 ## Set up the TimescaleDB extension
-When you have PostgreSQL and TimescaleDB installed, you can connect to it from
-your local system using the `psql` command-line utility. This is the same tool
-you might have used to connect to PostgreSQL before, but if you haven't
-installed it yet, check out our [installing psql][install-psql] section.
+
+When you have PostgreSQL and TimescaleDB installed, you can connect to it using
+the `psql` command-line utility.
+
+<CentOS />
 
 <procedure>
 
 ### Setting up the TimescaleDB extension
-1.  Initialize the database:
+
+1.  Enable and start the service:
+
     ```bash
-    /usr/pgsql-14/bin/postgresql-14-setup initdb
-    ```
-1. Enable and start the service:
-    ```
-    systemctl enable postgresql-14
-    systemctl start postgresql-14
+    sudo systemctl enable postgresql-14
+    sudo systemctl start postgresql-14
     ```
 
 1.  Connect to the PostgreSQL instance as the `postgres` superuser:
+
     ```bash
-    su postgres -c psql
+    sudo -u postgres psql
     ```
-    If your connection is successful, you see a message like this, followed
+
+    If your connection is successful, you see a message similar to this, followed
     by the `psql` prompt:
-    ```
-    psql (13.3, server 12.8 (Ubuntu 12.8-1.pgdg21.04+1))
-    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+
+    ```bash
+    could not change directory to "/root": Permission denied
+    psql (14.5)
     Type "help" for help.
-    tsdb=>
+
+    postgres=#
+
+    ```
+
+1.  Set the password for the `postgres` user using:
+
+    ```sql
+    postgres=#\password postgres 
+    ```
+
+1.  Exit from PostgreSQL using the command `\q` and use `psql` to connect:
```

Comment:
Should this be two steps?

Reply:
may be I should make it more specific and use `psql` client to connect?

Reply:
With the view of trying to keep the steps atomic, I think exiting from pg is one step, and using psql to connect is another.

Reply:
Yes @Loquacity !!! made it as two steps in the commit.